tuf/ngclient/updater.py

```diff
@@ -143,7 +139,7 @@ def refresh(self) -> None:
         self._load_snapshot()
         self._load_targets("targets", "root")
 
-    def get_one_valid_targetinfo(self, target_path: str) -> Dict:
+    def get_one_valid_targetinfo(self, target_path: str) -> Dict[str, Any]:
```

Comment:
I wonder if there is a better return type we could be using here? Rather than a dict which will only ever contain two key-value pairs where the values have different types?

I am not suggesting we change something here in this PR, just thinking out loud.

Reply:
Yeah, I think there is a common "willingness" and a bunch of issues related to improving this part of the API.
The suggestion in this one seems to be closest to yours: https://github.com/theupdateframework/tuf/issues/1411